zstash/update.py

```diff
@@ -107,8 +114,17 @@ def setup_update() -> Tuple[argparse.Namespace, str]:
         help="Hard copy symlinks. This is useful for preventing broken links. Note that a broken link will result in a failed update.",
     )
     args: argparse.Namespace = parser.parse_args(sys.argv[2:])
-    if args.hpss and args.hpss.lower() == "none":
+
+    if not args.hpss or args.hpss.lower() == "none":
         args.hpss = "none"
+        args.keep - True
```

Comment:
`= True`

Reply:
Ah!  That will make a difference!  :)  Good catch!